base/actions/actions.py

```diff
@@ -191,6 +191,15 @@ class HealthCheckProfileForm(BaseFormAction):
         "medical_condition",
     ]
 
+    MINOR_SLOTS = [
```

Comment:
Maybe a better name for this is `MINOR_SKIP_SLOTS`, because it's the slots we want to skip for minors? `MINOR_SLOTS` makes me think these are the slots we want for minors